src/pybind/mgr/dashboard/frontend/src/app/shared/models/permissions.ts

```diff
@@ -48,5 +49,6 @@ export class Permissions {
     this.grafana = new Permission(serverPermissions['grafana']);
     this.prometheus = new Permission(serverPermissions['prometheus']);
     this.nfs = new Permission(serverPermissions['nfs-ganesha']);
+    this.smb = new Permission(serverPermissions['nfs-ganesha']);
```

Comment:
btw, this is broken now. I assume the back-end is needed. Is it going to be done in the endpoints PR? @dnyanee1997 @Pegonzal 
```suggestion
    this.smb = new Permission(serverPermissions['smb']);
```

Reply:
Yes, it's already added here https://github.com/ceph/ceph/pull/60759/files#diff-e3e0967b6a6dc3d64b72b0742edf5d9626547be990d700994d6fb6c173eeca42

So as suggested, we just need to set 
`this.smb = new Permission(serverPermissions['smb']);` here

Reply:
Oh my bad,i was trying to test and forgot to remove.